docker-compose.yml

```diff
@@ -8,12 +8,23 @@ postgresql:
     - DB_EXTENSION=pg_trgm
   volumes:
     - /srv/docker/gitlab/postgresql:/var/lib/postgresql
+  networks:
+    - gitlab-tier
+
+redis:
+  restart: always
+  image: sameersbn/redis:latest
+  volumes:
+    - /srv/docker/gitlab/redis:/var/lib/redis
+  networks:
+    - gitlab-tier
+
 gitlab:
   restart: always
   image: sameersbn/gitlab:8.6.4
   links:
```

Comment:
yes. links are so yesterday. we need to get rid of them. Docker is in the process of retiring links and we should too :smile: